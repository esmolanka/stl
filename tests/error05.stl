
type Foo (a : Type) = a -> Unit
