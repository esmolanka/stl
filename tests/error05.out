tests/error05.stl:2:23-2:24: error:
    Covariant variable 'a' must not be used in contravariant positions.